networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -18,9 +18,20 @@ These release notes track the development of the External DNS Operator in {produ
 [id="external-dns-operator-release-notes-1.3.0"]
 == External DNS Operator 1.3.0
 
+The following advisory is available for the External DNS Operator version 1.3.0:
+
+*link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]
+
 This update includes a rebase to the 0.14.2 version of the upstream project.
 
-* link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[Kubernetes 0.14.2 Information]
+[id="external-dns-operator-.3.0-bug-fixes"]
+=== Bug fixes
+
+Previously, the ExternalDNS operand would not be deployable on HCP clusters. With this release, the Operand deploys in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])
```

Comment:
@jmanthei I think wordings of Andrey is right.
`Previously, the ExternalDNS Operator could not deploy operands on HCP clusters. With this release, the operator deploys operands in a running and ready state. (link:https://issues.redhat.com/browse/OCPBUGS-37059[*OCPBUGS-37059*])`

Reply:
okay, no problem. I will update.